python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -308,3 +382,48 @@ def _inplace_subtract(node: ResourceDict, resources: ResourceDict) -> None:
         assert k in node, (k, node)
         node[k] -= v
         assert node[k] >= 0.0, (node, k, v)
+
+
+def _inplace_add(a: Dict, b: Dict) -> None:
+    for k, v in b.items():
+        a[k] += v
```

Comment:
1) Can we use more meaningful variables other than a b k v?
2) Won't this cause a key error if k is not in a?